packages/appsync-emulator-serverless/lambdaRunner.js

```diff
@@ -1,39 +1,50 @@
 const log = require('logdown')('appsync-emulator:lambdaRunner');
+const { create: createUtils } = require('./util');
 
+const sendOutput = output => {
+  process.send({ type: 'success', output }, process.exit);
+};
 const sendErr = err => {
-  process.send({ type: 'error', error: err }, () => {
-    process.exit();
-  });
+  process.send({ type: 'error', error: err }, process.exit);
+};
+
+const run = async ({ lambda, context, payload }, callback) => {
+  const output = lambda(payload, context, callback);
+
+  if (!createUtils().isPromise(output)) return;
```

Comment:
`createUtils` was made for handling .vtl templates.  I do not think its the place to put `isPromise`